src/kava.js

```diff
@@ -117,6 +135,22 @@ class Kava extends BasePlugin {
       playTimeSum: 0.0,
       sessionStartTime: null
     });
+    this._performanceObserver.disconnect();
```

Comment:
to avoid of the duplicate code maybe put common reset and destroy actions in `_reset` method and call it from both destroy and reset.
```javascript
_reset(): void {
    this._timer.destroy();
    this._rateHandler.destroy();
    this._performanceObserver.disconnect();
    this._performanceEntries = [];
    this._pendingFragLoadedEvents = [];
}
```